modules/core/src/test/scala/org/scalasteward/core/nurture/NurtureAlgTest.scala

```diff
@@ -0,0 +1,73 @@
+package org.scalasteward.core.nurture
+
+import munit.CatsEffectSuite
+import org.http4s.HttpApp
+import org.http4s.dsl.Http4sDsl
+import org.scalasteward.core.TestInstances._
+import org.scalasteward.core.TestSyntax._
+import org.scalasteward.core.data.{DependencyInfo, RepoData, UpdateData}
+import org.scalasteward.core.edit.EditAttempt.UpdateEdit
+import org.scalasteward.core.git.{Branch, Commit}
+import org.scalasteward.core.mock.MockContext.context.nurtureAlg
+import org.scalasteward.core.mock.{MockEff, MockState}
+import org.scalasteward.core.repoconfig.RepoConfig
+import org.scalasteward.core.vcs.data.{NewPullRequestData, Repo}
+
+class NurtureAlgTest extends CatsEffectSuite with Http4sDsl[MockEff] {
```

Comment:
I always bothered me that the heart of steward the `NurtureAlg` was not tested at all. Well done.

Reply:
We're slowly getting there. Unfortunately there is a lot of stuff to set up until we can meaningfully test `NurtureAlg`.